n_t/Snakefile

```diff
@@ -132,25 +142,25 @@ rule sp_download:
 
 rule run_stairwayplot:
     input:
-        expand(output_dir + "/{seeds}/{chrms}.trees",
+        expand(output_dir + "/Intermediate/{seeds}/{chrms}.trees",
             chrms=chrm_list, seeds=seed_array),
         stairwayplot_code,
-    output: output_dir + "/{seeds}/stairwayplot_estimated_Ne.txt"
+    output: output_dir + "/Results/{seeds}/stairwayplot_estimated_Ne.txt"
     threads: 20
     run: 
-        inputs = expand(output_dir + "/{seeds}/{chrms}.trees",
+        inputs = expand(output_dir + "/Intermediate/{seeds}/{chrms}.trees",
             seeds=wildcards.seeds, chrms=chrm_list)
         runner = stairway.StairwayPlotRunner(
             workdir= output_dir + "/stairwayplot/" + wildcards.seeds+"/",
             stairway_dir=pathlib.Path.cwd() / "stairwayplot")
-        runner.ts_to_stairway(inputs, num_bootstraps=12)
+        runner.ts_to_stairway(inputs, num_bootstraps=200)
```

Comment:
200 seems like a good default, but can will pull this out to the config file as well?

Reply:
Sure.

Reply:
if you make that small change and commit, let's merge